DirectEthernet/README.txt

```diff
@@ -0,0 +1,12 @@
+The direct ethernet is no longer being actively developed here. To
```

Comment:
Maybe makes these `.md` files. Github renders them nicely (with links, etc) and they're still perfectly readable as plain text.